lib/ipmi_backend_utils.pm

```diff
@@ -98,11 +98,14 @@ sub setup_console_in_grub {
 
     #set grub config file
     my $grub_cfg_file;
+    my $grub_default_file;
```

Comment:
This grub_default_file seems actually useless when you choose the way to directly change grub.cfg?

Reply:
$grub_default_file will also be uploaded along with $grub_cfg_file